Content.Client/UserInterface/HandsGui.cs

```diff
@@ -36,6 +40,8 @@ public HandsGui()
         {
             IoCManager.InjectDependencies(this);
 
+            _configManager.OnValueChanged(CCVars.HudTheme, UpdateHudTheme, invokeImmediately: true);
```

Comment:
Read my review above, you should correct the value if it's incorrect in `UpdateHudTheme`